free_fleet_adapter/free_fleet_adapter/fleet_adapter.py

```diff
@@ -272,7 +272,7 @@ def main(argv=sys.argv):
     parser.add_argument('-sim', '--use_sim_time', action='store_true',
                         help='Use sim time, default: false')
     parser.add_argument(
-        '--zenoh-config',
+        '--zenoh_config',
```

Comment:
`argparse` handles the translation from dash to underscore, this change is not needed. This comment applies to all other changes to arguments below

just as a reference, command line arguments should use dashes in general (POSIX-style), since underscores may be mistaken as a an empty space on a terminal